src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -569,12 +569,6 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('remove_repos', options_list=['--remove'], nargs='*',
                    help='repository permissions to be removed from the targeted connected registry and it\'s succesors sync scope maps. Use the format "--remove [REPO1 REPO2 ...]" per flag. ' + repo_valid_actions)
 
-    with self.argument_context('acr connected-registry repo') as c:
-        c.argument('add_repos', options_list=['--add'], nargs='*',
-                   help='repository permissions to be added to the targeted connected registry and it\'s ancestors sync scope maps. Use the format "--add [REPO1 REPO2 ...]" per flag. ' + repo_valid_actions)
-        c.argument('remove_repos', options_list=['--remove'], nargs='*',
-                   help='repository permissions to be removed from the targeted connected registry and it\'s succesors sync scope maps. Use the format "--remove [REPO1 REPO2 ...]" per flag. ' + repo_valid_actions)
```

Comment:
Please add `deprecate_info` first to deprecate them, and then remove them in the breaking change window of next sprint

Reply:
The entire command was already removed.

Reply:
What I mean is that these commands are more recommended to be removed in the breaking change window of the next sprint, otherwise it may cause unexpected damage to the customers' automation scripts. You can read this guideline [breaking_changes](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azcli_docs/breaking_changes) to learn more about CLI breaking change process

Reply:
thank you I'll remove them. I wasn't aware of this new policy. I thought it'd be fine for this command since they were only available for the month of releasee of the feature 2 years ago.